worlds/sc2/Items.py

```diff
@@ -1322,22 +1322,54 @@ def get_full_item_list():
-    ItemNames.INFESTOR: ItemData(8 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 8, SC2Race.ZERG, classification=ItemClassification.progression, origin={"hots"}),
-    ItemNames.ULTRALISK: ItemData(9 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 9, SC2Race.ZERG, classification=ItemClassification.progression, origin={"hots"}),
-    ItemNames.SPORE_CRAWLER: ItemData(10 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 10, SC2Race.ZERG, classification=ItemClassification.progression, origin={"hots"}),
-    ItemNames.SPINE_CRAWLER: ItemData(11 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 11, SC2Race.ZERG, classification=ItemClassification.progression, origin={"hots"}),
-    ItemNames.CORRUPTOR: ItemData(12 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 12, SC2Race.ZERG, classification=ItemClassification.progression, origin={"ext"}),
-    ItemNames.SCOURGE: ItemData(13 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 13, SC2Race.ZERG, classification=ItemClassification.progression, origin={"bw", "ext"}),
-    ItemNames.BROOD_QUEEN: ItemData(14 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 4, SC2Race.ZERG, classification=ItemClassification.progression, origin={"bw", "ext"}),
-    ItemNames.DEFILER: ItemData(15 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 14, SC2Race.ZERG, classification=ItemClassification.progression, origin={"bw"}),
+    ItemNames.ZERGLING: ItemData(0 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 0, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="Fast inexpensive melee attacker. Can morph into a Baneling."),
+    ItemNames.SWARM_QUEEN: ItemData(1 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 1, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="Ranged support caster. Can use the Spawn Creep Tumor and Rapid Transfusion abilities."),
+    ItemNames.ROACH: ItemData(2 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 2, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="Durable short ranged attacker. Regenerates life quickly when burrowed."),
+    ItemNames.HYDRALISK: ItemData(3 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 3, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="High-damage generalist ranged attacker."),
+    ItemNames.ZERGLING_BANELING_ASPECT: ItemData(4 + SC2HOTS_ITEM_ID_OFFSET, "Morph", 5, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="Anti-ground suicide unit. Does damage over a small area on death."),
+    ItemNames.ABERRATION: ItemData(5 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 5, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="Durable melee attacker that deals heavy damage."),
```

Comment:
Optional: Is it worth mentioning they're towering? (ie they can walk over other units, which is where a lot of their DPS potential comes from)